stack.yaml

```diff
@@ -41,3 +41,8 @@ extra-deps:
   - primitive-unlifted-0.1.3.0
   - run-st-0.1.1.0
   - tuples-0.1.0.0
+
+nix:
+  enable:     true
+  pure:       true
+  shell-file: shell.nix
```

Comment:
The [Stack/Nix integration](https://docs.haskellstack.org/en/stable/nix_integration/) has really improved!

Reply:
Totally! this is awesome.

Do we want to bump stack to LTS-17 here & now? or are there issues still with it?

Reply:
Fair, yeah we may as well. I have that in a different branch, so I didn't think about it. Will do now 👍

Reply:
Will see if that builds ⚙️ My other branch has a couple of these pinned dependencies updated as well.